src/frontend/src/pages/FinancePage/ReimbursementRequestDetailPage/SubmitToSaboModal.tsx

```diff
@@ -58,8 +58,10 @@ const SubmitToSaboModal = ({ open, setOpen, reimbursementRequest }: SubmitToSabo
       open={open}
       onHide={() => setOpen(false)}
       title="Input these fields into the Sabo Form"
-      cancelText="Cancel"
-      submitText="Submit to Sabo"
+      cancelText={isReimbursementRequestSaboSubmitted(reimbursementRequest) ? '' : 'Cancel'}
```

Comment:
make isReimbursementRequestSaboSubmitted(reimbursementRequest) a variable and just call it here